javascript/javascript2/week2/hyfBayHelpers.js

```diff
@@ -0,0 +1,98 @@
+/* DONT MODIFY ANY OF THIS CODE!!!*/
+
+window.availableCountries = [
+  "Denmark",
+  "Sweden",
+  "Norway",
+  "Germany",
+  "Iceland",
+  "England",
+];
+
+window.getAvailableProducts = function () {
+  function getRandomInt(min, max) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  }
+
+  function getRandomItem(availableProductNames) {
+    return availableProductNames[
+      getRandomInt(0, availableProductNames.length - 1)
+    ];
+  }
+
+  function getRandomProductname() {
+    const preWords = [
```

Comment:
These 2 consts would be defined anew every time the function is called. There is no need to do that so you can move them out ouf the function. 
It is typical to define unchanging constants up top, some also use the convention to have their variable names in all caps in snake case - PRE_WORDS or PRODUCT_NAMES.